DEVELOPING.md

```diff
@@ -34,8 +32,19 @@
    - _Suggestion:_ Use branch name format of `<initials>/<work-title>`.
      - Example: `mb/refactor-tests`
 1. Make code changes and build: `yarn build`
-1. Run changed commands: e.g., `./bin/run.js my-topic my-command`
-1. Write tests and run: `yarn test` ([unit](#unit-tests)) and/or `yarn test:nuts` ([NUTS](#nuts-non-unit-tests))
+1. Update your MCP client to launch the local build with node instead of the npm package via npx, example with vscode MCP config file:
+```json
+{
+  "servers": {
+    "salesforce": {
+      "command": "node",
+      "args": ["/full/path/to/mcp/lib/index.js", "--toolsets", "all", "--orgs", "ALLOW_ALL_ORGS"]
+    }
+  }
+}
+```
+1. Start the MCP server and call the tools via prompts
+1. Write tests and run: `yarn test` ([unit](#unit-tests))
 1. Show all changed files: `git status`
 1. Add all files to staging: `git add .`
 1. Commit staged files with helpful commit message: `git commit`
```

Comment:
The TODO here? I know you didnt add it, but i wonder if it was going to mention conventional commits syntax?